websites/F/FL Studio/presence.ts

```diff
@@ -0,0 +1,217 @@
+    presenceData.details = 'Viewing FLEX'
+    presenceData.state = 'Free FL Studio Plugin'
+    matched = true
+  }
+  // MIDI Controllers
+  else if (pathname.includes('/midi-controllers')) {
+    presenceData.details = 'Viewing MIDI Controllers'
+    presenceData.state = 'Hardware for FL Studio'
+    matched = true
+  }
+  // Artists/Power Users
+  else if (pathname.includes('/artists')) {
+    presenceData.details = 'Viewing FL Studio Artists'
+    presenceData.state = 'Famous Producers Using FL Studio'
+    matched = true
+  }
+  // Buy/Purchase pages
+  else if (pathname.includes('/buy') || pathname.includes('/purchase')) {
+    presenceData.details = 'Purchasing FL Studio'
+    presenceData.state = 'Choose Your Edition'
+    matched = true
+  }
+  // Release notes
+  else if (pathname.includes('/release')) {
+    presenceData.details = 'Reading Release Notes'
+    presenceData.state = 'What\'s New in FL Studio'
+    matched = true
+  }
+  // Specials/Offers
+  else if (pathname.includes('/specials')) {
+    presenceData.details = 'Viewing Special Offers'
+    presenceData.state = 'Limited Time Deals'
+    matched = true
+  }
+  // Forum
+  else if (pathname.includes('forum')) {
+    presenceData.details = 'Browsing FL Studio Forum'
+    presenceData.state = 'Community Discussions'
+    matched = true
+  }
+  // Support pages
+  else if (pathname.includes('/support')) {
+    presenceData.details = 'Getting Support'
+    presenceData.state = 'Help & FAQ'
+    matched = true
+  }
+  // Contact
+  else if (pathname.includes('/contact')) {
+    presenceData.details = 'Contact Information'
+    presenceData.state = 'Get in Touch with Image-Line'
+    matched = true
+  }
+  // Jobs
+  else if (pathname.includes('/jobs')) {
+    presenceData.details = 'Viewing Job Opportunities'
+    presenceData.state = 'Careers at Image-Line'
+    matched = true
+  }
+  // Press
+  else if (pathname.includes('/press')) {
+    presenceData.details = 'Press Information'
+    presenceData.state = 'Media Resources'
+    matched = true
+  }
+  // Legal pages
+  else if (pathname.includes('/legal')) {
+    presenceData.details = 'Legal Information'
+    presenceData.state = 'Terms & Privacy'
+    matched = true
+  }
+
+  else if (pathname === '/learn' || pathname.startsWith('/learn/')) {
+    presenceData.details = 'Learning FL Studio'
+    presenceData.state = 'FL Studio Academy'
+    matched = true
+  }
+
+  // Default fallback if no specific page matched
+  if (!matched) {
+    presenceData.details = strings.browse
+    presenceData.state = 'FL Studio Website'
+  }
+
+  // Apply privacy mode
+  if (privacyMode) {
+    delete presenceData.state
+    presenceData.details = strings.browse
+  }
+
+  // Apply timestamp setting
+  if (!showTimestamp || privacyMode) {
+    delete presenceData.startTimestamp
+  }
+
+  // Set the activity
+  if (presenceData.details) {
+    presence.setActivity(presenceData)
+  }
+  else {
+    presence.setActivity()
```

Comment:
If you want to hide the activity rather than show a blank activity, use `presence.clearActivity()`